docs/daemon/configuration/genesis.mdx

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_position: 2
+sidebar_label: "Genesis"
+id: daemon-genesis
+title: Genesis Configuration
+description: Genesis specification
+slug: /daemon/genesis
+---
+
+A Genesis file defines the initial state and parameters needed to bootstrap a new blockchain network. It serves as the starting point for the entire blockchain system by defining key aspects of the network, including its initial validators, initial height, consensus rules, and more. All the nodes in the network need to have the same genesis file to start with the same initial conditions. Any discrepancies or errors in the genesis file could prohibit the node either from joining the network or from producing blocks.
+
+## Genesis file with minimal configuration
+
+Here is an example of a genesis.json file with minimal configuration:
+
+```json title="sample genesis.json file"
+{
+  "genesis_time": "2023-08-28T22:24:04.513472Z",
+  "chain_id": "kwil-chain-RQMl2F",
+  "initial_height": "0",
+  "validators": [
+    {
+      "address": "6F129A1818CB42DD80FFE51943EAC0BC9FF9E4EE",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "L737LDRTYv7P5I2apSxOBwJD3U0CS7b2eo9K37s2lPM="
+      },
+      "power": "1",
+      "name": "validator-0"
+    }
+  ],
+  "app_hash": "4F8735846B8CFB01A277430A392DD3DB7B66510C358C5F4A2AF88A348CB20A62"
+}
+```
+
+## Genesis file contents
+
+A Genesis file essentially consists of:
+
+### Genesis time
+
+ `genesis_time`: This field specifies the official time at which the blockchain is initialized.
+
+### Chain ID
+
+ `chain_id`: This field specifies the ID of the blockchain network. This must be unique for every blockchain. The ChainID must be less than 50 symbols.
+
+### Initial height
+
+ `initial_height`: This field sets the initial height the blockchain should begin.  `Default value is 0`, but in the scenario of network upgrade or restart, starting from the stopped height ensures uniqueness to previous heights.
```

Comment:
Why is `Default value is 0` in backticks?  Also please pay attention to grammar in the second sentence